web/backend_api_builder.py

```diff
@@ -78,3 +78,9 @@ def case_id(self, case_id):
         :return: case endpoint
         """
         return self.cases + '/{}'.format(case_id)
+
+    def case_list(self,id):
```

Comment:
you can add explicit functions here ( case_list_for_beneficiery(ben_id) and case_list_for_members(mem_id)